config/project.yaml

```diff
@@ -475,6 +475,31 @@ fields:
     translationKeyFormat: null
     translationMethod: language
     type: craft\fields\PlainText
+  3cfdb7b7-949e-4841-b88d-148f1ea0b289:
+    name: 'Child page display'
+    handle: childPageDisplay
+    instructions: "How should child pages of this page be displayed?\n\nNote: photo grids will only display if each child page has a hero image."
+    searchable: false
+    translationMethod: none
+    translationKeyFormat: null
+    type: craft\fields\Dropdown
+    settings:
+      optgroups: true
+      options:
+        -
+          label: 'Do not show child pages'
+          value: none
+          default: '1'
```

Comment:
Couple of questions, should this be the default value? And do we need to give a choice of style for child pages? Could we work it out based on if all children have hero images as before, making this a "show child pages" option?

Reply:
This makes sense, happy to go with this then :+1: